exporter/importables.go

```diff
@@ -68,6 +68,10 @@ var (
 		"DBC":        ".dbc",
 		"R_MARKDOWN": ".Rmd",
 	}
+	metastorePrivilegesToAdd = []string{`CREATE_CATALOG`, `CREATE_CONNECTION`,
+		`CREATE_EXTERNAL_LOCATION`, `CREATE_PROVIDER`, `CREATE_RECIPIENT`, `CREATE_SCHEMA`,
+		`CREATE_STORAGE_CREDENTIAL`, `SET_SHARE_PERMISSION`}
+	defaultPrivilegesToAdd = []string{`ALL_PRIVILEGES`}
```

Comment:
maintaining a separate list of `CREATE_X` privileges mapping would be sensible, as we do not add new securables that often